process/subprocesses/_04_create_population_grid.py

```diff
@@ -74,7 +74,7 @@ def derive_population_grid_variables(r):
     SELECT h."grid_id",
             COUNT(i.*) intersection_count
     FROM {r.config["population_grid"]} h
-    LEFT JOIN {r.config["intersections_table"]} i
+    LEFT JOIN {r.config["intersections_pedestrian_table"]} i

```

Comment:
I'm not necessarily convinced that it is worthwhile evaluating cycling intersection connectivity as distinct from pedestrian connectivity...  Is this something that people use and are interested in? if its not, I'd be inclined to avoid the extra complication.